cypress/integration/manualTests/basic-enduser.ts

```diff
@@ -38,58 +111,77 @@ function testWorkflow(url: string, version1: string, version2: string, trsUrl: s
     cy.url().should('contain', '?tab=tools');
   });
 
-  // This tends to be flaky so we are leaving it out for now
-  //
-  // it('DAG tab works', () => {
-  //   goToTab('DAG');
-  //   cy.url().should('contain', '?tab=dag');
-  //   cy.get('[data-cy=dag-holder]');
-  // });
+  it('DAG tab works', () => {
+    cy.contains('.mat-tab-label', 'DAG').click();
+    cy.url().should('contain', '?tab=dag');
+    cy.get('[data-cy=dag-holder]');
+  });
 
   let launchWithTuples: any[] = [];
   if (type === 'WDL') {
     it('get the svg icons', () => {
-      cy.get('[data-cy=dnanexusIcon]').within(() => {
-        cy.get('svg').should('exist');
+      cy.get('[data-cy=dnanexusLaunchWith]').within(() => {
+        cy.get('img').should('exist');
       });
-      cy.get('[data-cy=terraIcon]').within(() => {
-        cy.get('svg').should('exist');
+      cy.get('[data-cy=terraLaunchWith]').within(() => {
+        cy.get('img').should('exist');
       });
-      cy.get('[data-cy=anvilIcon]').within(() => {
-        cy.get('svg').should('exist');
+      cy.get('[data-cy=anvilLaunchWith]').within(() => {
+        cy.get('img').should('exist');
       });
     });
   }
   if (type === 'CWL') {
     launchWithTuples = [
       // pairs of [launch button text, expected href]
-      ['CGC', '?trs=' + trsUrl],
-      ['NHLBI BioData Catalyst', '?trs=' + trsUrl],
+      ['cgcLaunchWith', '?trs=' + trsUrl],
+      ['nhlbiLaunchWith', '?trs=' + trsUrl],
+      ['cavaticaLaunchWith', '?trs=' + trsUrl],
     ];
   } else if (type === 'WDL') {
     launchWithTuples = [
       // pairs of [launch button text, expected href]
-      ['DNAstack', '?descriptorType=wdl&path=' + url],
-      ['DNAnexus', '?source=' + trsUrl],
-      ['Terra', url + ':' + version2],
-      ['AnVIL', url + ':' + version2],
-      ['NHLBI BioData Catalyst', url + ':' + version2],
+      ['dnanexusLaunchWith', '?source=' + trsUrl],
+      ['terraLaunchWith', url + ':' + version2],
+      ['anvilLaunchWith', url + ':' + version2],
+      ['nhlbiLaunchWith', url + ':' + version2],
     ];
+  } else if (type === 'NFL') {
+    launchWithTuples = [['nextflowtowerLaunchWith', url + '&revision=' + version2]];
```

Comment:
Would be nice if Galaxy were here too, but I don't want to introduce feature creep. If complicated, could you create a follow up ticket and/or add it the doc for manual testing.

Reply:
Created a followup ticket to add smoke testing for both Galaxy and DNA Stack, https://github.com/dockstore/dockstore/issues/4278. They both have a different structure than the other buttons (and I'm not seeing any default options available when running locally... so there may be more setup on my end).